src/main/java/com/eerussianguy/blazemap/feature/maps/WorldMapGui.java

```diff
@@ -358,6 +380,11 @@ public boolean keyPressed(int key, int x, int y) {
             return true;
         }
 
+        if(key == GLFW.GLFW_KEY_F12) {
+            AtlasExporter.export(new AtlasExporter.Task(this.dimension, this.getMapType().getID(), this.mapRenderer.getVisibleLayers()));
+            return true;
+        }
```

Comment:
IIRC, we're actually doing the key listening wrong based on the Forge docs. But that's a bigger thing we can fix later. It's fine to continue the same as we've been doing for now.